Automattic/woocommerce-payments
# Remove 'View dispute' link in Transaction Timeline

Fixes #6928 

#### Changes proposed in this Pull Request

* Remove 'View dispute' link in the timeline of transaction details page.
* Since removing the 'View dispute' link, some e2e tests are broken. I skip those tests for now. Created https://github.com/Automattic/woocommerce-payments/issues/7377 to re-enable and fix them.

#### Testing instructions

* Purchase a product with `4000000000000259` to trigger a dispute upon purchase.
* View the dispute by going to Payments > Disputes > click on the newly created dispute. You will be redirected to the transaction details page.
* In the timeline of the transaction details page, with base branch, you will see a 'View dispute' link.

![Screenshot 2023-08-04 at 09 27 23](https://github.com/Automattic/woocommerce-payments/assets/73803630/d28ee056-ac01-4809-b0cc-0ed43127d084)

With PR branch, you shouldn't see the link.

-------------------


**Post merge**
